lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -262,6 +267,32 @@ def tokenize_apple_pay_token(apple_pay_payment_token, options = {})
         end
       end
 
+      def tokenize_apple_google(payment, options = {})
+        if payment.inspect.include?('google_pay')
```

Comment:
I would check `payment.source == :google_pay` directly here, rather than `.include?`

There is also some possibility the `source` will be something other than these two (`android_pay` or `network_token`), which we should handle/guard against with a guard clause or a third case.